W-15712506 – Update configure-horizontal-autoscaling.adoc (#1281)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/configure-horizontal-autoscaling.adoc

- --balancing-ignore-label=vpc.amazonaws.com/eniConfig
----

> These configuration recommendations serve as a guideline; however, the actual configuration of the cluster autoscaler (CA) depend on the cloud or infrastructure provider (vendor) you use and the relevant topology considerations. It is your responsibility to configure the CA appropriately.

Reviewer: ```suggestion
These configuration recommendations serve as a guideline. However, the actual configuration of the cluster autoscaler (CA) depends on the cloud or infrastructure provider (vendor) you use and the relevant topology considerations. You must configure the CA appropriately.
```

Reviewer: The sentence had more than 32 words, so I split the sentence in two.

        rtf.mulesoft.com/id: "{{ .Values.id }}"
----

> The recommendation to use `whenUnsatisfiable: DoNotSchedule` ensures that pods are evenly spread across availability zones. To modify this setting, use the K8S templates CRD to adjust your deployments using HPA to `whenUnsatisfiable: DoNotSchedule`. You can also use dedicated namespaces (environments) to separate the autoscaling workloads from the classic ones. If you do not make this change, you use `whenUnsatisfiable: ScheduleAnyway`.

Reviewer: ```suggestion
Using `whenUnsatisfiable: DoNotSchedule` guarantees that pods are evenly spread across availability zones. To modify this setting, use the K8S templates CRD to adjust your deployments using HPA to `whenUnsatisfiable: DoNotSchedule`. You can also use dedicated namespaces (environments) to separate the autoscaling workloads from the classic ones. If you don't make this change, you use `whenUnsatisfiable: ScheduleAnyway`.
```

Enabling HPA can result in customers incurring additional flow usage when your application scales horizontally. To avoid overages from unpredicted scaling, configure the maximum configured replicas judiciously to stay within purchased flow limits. Track your incurred flow usage through xref:general::usage-reports.adoc[usage reports].

HPA uses CPU utilization at average of 70% +
HPA bases on https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/#how-does-a-horizontalpodautoscaler-work[resource requests].
[quote,kubernetes.io]
----
When target utilization value is set, the controller calculates the utilization value as a percentage of the equivalent resource request on the containers in each pod.
----

> If pod average utilization is greater than 70% in a period (periods for scale up are set as 180 seconds) HPA will scale up. Refer to the Kubernetes documentation for additional details.

Reviewer: ```suggestion
If pod average use is greater than 70% in a period (periods for scale-up are set as 180 seconds), HPA scales up. Refer to the Kubernetes documentation for additional details.
```

Reviewer: Do you have a link to the Kubernetes documentation the reader should go to?
